tensorflow/core/kernels/mlir_generated/BUILD

```diff
@@ -992,6 +1039,27 @@ gpu_kernel_library(
     unroll_factors = "4",
 )
 
+gpu_kernel_library(
+    name = "gpu_sub_kernels_experimental",
+    op = "sub",
+    tile_size = "1024",
+    jit_i64_indexed_for_large_tensors_types = [
```

Comment:
Can we leave the `jit_types` untouched for now not to grow TF's binary size? 
Meaning, "i8", "i16", "ui8", and "ui16" would only be in `jit_types`

Reply:
Added the 'jit_types' back.